src/components/TopBar/TopBarStyles.js

```diff
@@ -0,0 +1,71 @@
+import styled from 'styled-components';
+
+export const StyledTopBarWrapper = styled.nav`
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 100%;
+  height: 57px;
+  z-index: 2;
+`;
+
+export const StyledTopBar = styled.div`
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 100%;
+  height: 56px;
+  background-color: #2a2a2a;
+  box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.12), 0px 2px 4px rgba(0, 0, 0, 0.14),
+    0px 1px 2px rgba(0, 0, 0, 0.2);
+  z-index: 3;
+`;
+
+export const StyledHeader = styled.header`
+  position: absolute;
+  width: 176px;
+  height: 24px;
+  left: 69px;
+  top: 16px;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 19px;
+  line-height: 22px;
+  display: flex;
+  align-items: center;
+  color: #3c78c8;
+`;
+
+export const StyledControls = styled.div`
+  position: absolute;
+  width: 99px;
+  height: 24px;
+  right: 16px;
+  top: 16px;
```

Comment:
`  height: 24px;` duplicates `StyledHeader` height.
` top: 16px;` duplicates `StyledHeader` top.

I think it should be one number

![image](https://user-images.githubusercontent.com/6686679/82830833-82493780-9ebf-11ea-8319-54881d35de93.png)

Reply:
What if you just set `padding-top: 24px` for the `StyledTopBar`?
And use a usual position instead of absolute for header and controls.
It can be a flex container or `display: inline-block` or `float: right`.

Reply:
What do you think is better / easier in this case:
- use `position: absolute` for the `NeuralNotes` and the icons (like we do now0
- or use `flex` or `display: inline` or `float: right` to implement this layout

Reply:
For some reason I got a feeling that using `position: absolute` is not the right way for this case.
But maybe just because I used to not use `absolute` 🤔 
Your approach doesn't look too bad :)

Reply:
I agree with you, so I'll change this to `display: flex`. Only that I ask for it's to allow make margin- or padding-left for logo)

Reply:
>.Only that I ask for it's to allow make margin- or padding-left for logo)

What do you mean? :)

Reply:
![e853ffe456](https://user-images.githubusercontent.com/19979138/82950412-ef3afb00-9fad-11ea-9406-68f42a97d679.jpg)

I mean this distance. It must be reserved for further "back-button" 

![image](https://user-images.githubusercontent.com/19979138/82950696-6a041600-9fae-11ea-95b5-e62a484e8b1f.png)

Reply:
Oh, okay sure padding is fine :)